submissions/NastjonkaK/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,116 @@
+	            			<img src="Icons/cloud.png">
+	            			<div class="captions"> Cloud </div>
+	            		</a>
+            		</li>
+            		<li class="popupIconsItem">
+	            		<a href="#">
+	            			<img src="Icons/store.png">
+	            			<div class="captions"> Store </div>
+	            		</a>
+            		</li>
+            		<li class="popupIconsItem">
+	            		<a href="#">
+	            			<img src="Icons/translate.png">
+	            			<div class="captions"> Translate </div>
+	            		</a>
+            		</li>
+            		<li class="popupIconsItem">
+	            		<a href="#">
+	            			<img src="Icons/music.png">
+	            			<div class="captions"> Music </div>
+	            		</a>
+            		</li>
+            		<li class="popupIconsItem">
+	            		<a href="#">
+	            			<img src="Icons/movie.png">
+	            			<div class="captions"> Movie </div>
+	            		</a>
+            		</li>
+            		<li class="popupIconsItem">
+	            		<a href="#">
+	            			<img src="Icons/alarm.png">
+	            			<div class="captions"> Alarm </div>
+	            		</a>
+            		</li>
+            		<li class="popupIconsItem space"></li>
+            	</ul>
+            		<input type="checkbox" id="button_more" class="MoreApps">
+            		<label for="button_more" class="MorePopUp"> More </label>
+            		<hr class="line">
+            	<ul class="hiddenBlock">
+            		<li class="popupIconsItem hidden">
+	            		<a href="#">
+	            			<img src="Icons/book.png">
+	            			<div class="captions"> Books </div>
+	            		</a>
+            		</li>
+            		<li class="popupIconsItem hidden">
+	            		<a href="#">
+	            			<img src="Icons/notes.png">
+	            			<div class="captions"> Notes </div>
+	            		</a>
+            		</li>
+            		<li class="popupIconsItem hidden">
+	            		<a href="#">
+	            			<img src="Icons/photo.png">
+	            			<div class="captions"> Photo </div>
+	            		</a>
+            		</li>
+           		<li class="popupIconsItem hidden">
+	            		<a href="#">
+	            			<img src="Icons/timer.png">
+	            			<div class="captions"> Timer </div>
+	            		</a>
+            		</li>
+            		<li class="popupIconsItem hidden">
+	            		<a href="#">
+	            			<img src="Icons/mail.png">
+	            			<div class="captions"> Mail </div>
+	            		</a>
+            		</li>
+            		<li class="popupIconsItem hidden">
+	            		<a href="#">
+	            			<img src="Icons/chat.png">
+	            			<div class="captions"> Chat </div>
+	            		</a>
+            		</li>
+            	</ul>
+            </div>
+			<li class="menu-item"> <img src="Icons/bell.png"></li>
+			<li class="menu-item"> <img id="ava" src="Icons/avatar.png"></li>
+			<div id="triangle">
```

Comment:
Also, more "canonical" is useing `::after` or `::before` css pseudo-classes for such porposes.

Reply:
I don't think it's appropriate in this case.